### ISSUE TITLE
Email links are confusing and/or broken

### ISSUE THREAD

**Issue by crenshaw-dev** [caprivacy/caprivacy#68]
In the full view (https://caprivacy.github.io/caprivacy/full/) the email links seem to point to the base page rather than anything meaningful.

At first glance, the underlying data doesn't seem to contain email addresses for the companies.

**Comment by rkiddy** [comment]
Yes, I was expecting to see email addresses in the companies.csv file, but there are none.

Where are they supposed to be? It would be easiest to have an e-mail column next to the url in the csv file.

I would add the data brokers in CA, but I want to make sure the schema is clear so I do not cause problems.
